src/pretix/multidomain/templatetags/eventurl.py

```diff
@@ -45,11 +45,13 @@ def render(self, context):
             for k, v in self.kwargs.items()
         }
         view_name = self.view_name.resolve(context)
-        event = self.event.resolve(context)
+        event = self.event.resolve(context) if self.event is not False else False
         url = ''
         try:
             if self.absolute:
```

Comment:
It took me a few minutes too understand all the different cases this can be called with now… if you have time… would this be worth some tests?

Reply:
Added tests and improved the docstring